processor/src/operations/horner_ops.rs

```diff
@@ -287,12 +330,20 @@ mod tests {
 
         let alpha = QuadFelt::new(alpha_mem_word[0], alpha_mem_word[1]);
 
-        let acc_new = stack_state
+        let tmp = stack_state
```

Comment:
nit: although this corresponds nicely with the docs, it's not the most informative name when looking at the code in isolation. So maybe renaming it to `tmp_acc` both here and in docs?

Reply:
Makes sense